romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -1,6 +1,5 @@
 %YAML 1.1
```

Comment:
We'll need to get the schema tester running on these.  We need to get the tests running on Travis, too.  I'll file an issue for both of those tasks.

Reply:
Done.